### PR TITLE
feat: add definitions and modify discover

### PR DESCRIPTION
**Pull request by lluisCM**
Resolves 

- [ ] I have added automatic tests where applicable.
- [ ] The PR contains a description of what has been changed.
- [ ] The description contains manual test instructions.
- [ ] The PR contains updates to the release notes.
- [ ] I have verified that the documentation is still up to date.

## Changes

All definitions are moved to each integration, schemas are in pipeline. Collection and validation of the definitions happen on host.

### CODE REVIEW

File: resource/definitions/opener/camera-nuke-opener.json.orig [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
```
Comment (henriknorin-ftrack) [inline resource/definitions/opener/camera-nuke-opener.json.orig]: Seems to be a bunch of these .orig files around in the PR?

File: hook/discover_nuke.py [hunk 2]
```diff
@@ -44,8 +44,15 @@ def on_launch_nuke_pipeline(session, event):
 
     nuke_bootstrap_plugin_path = os.path.join(nuke_bootstrap_path, 'plugins')
```
Comment (henriknorin-ftrack) [inline hook/discover_nuke.py:45]: Remove if not used.

**Review by henriknorin-ftrack — CHANGES_REQUESTED** [review]
Please remote the .orig files from PR

File: resource/definitions/opener/camera-nuke-opener.json.orig (hunk 1 above)
Comment (lluisCM) [reply resource/definitions/opener/camera-nuke-opener.json.orig]: removing them now

File: hook/discover_nuke.py (hunk 2 above)
Comment (lluisCM) [reply hook/discover_nuke.py:45]: I like to leave those, If the user wants to add a plugin it should be added in there, and as all the other integrations it should be able to be read from there automatically.

**Review by henriknorin-ftrack — APPROVED** [review]

**Pull request merged**